### PR TITLE
Feature/upgrade ethers.js

### PR DESCRIPTION
**Pull request by jsidorenko**

### CODE REVIEW

File: index.thread.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 // @flow
 import { self } from 'react-native-threads';
-import { HDNode } from 'ethers';
+import { utils } from 'ethers';
+
+const { HDNode } = utils;
```
Comment (alazarevski) [inline index.thread.js]: Note: the bundles should be updated on the ios/android app
yarn build-thread-ios
yarn build-thread-android

**Comment by grilix** [comment]
@jsidorenko do we need further changes here or we are waiting on something before we merge it?

**Comment by jsidorenko** [comment]
@grilix I think Vitor wanted to have a release candidate first and then merge this PR into the next release

**Comment by jsidorenko** [comment]
Closed in favour of https://github.com/pillarwallet/pillarwallet/pull/1508

**Pull request closed without merging**